src/components/dashboard/components/Dashboard.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 
-import { TEST } from '../constants';
+import sideMenu from '../../sideMenu';
+
+const { SideMenu } = sideMenu.components;
```

Comment:
This can further be destructured too. `const { components: { SideMenu } } = sideMenu`

Reply:
Alright, on it.